m4_sessions/1808-inning/Perspective_to_succeed

```diff
@@ -1,19 +1,63 @@
 ## Group Members:
+john livingston
+spenser leighton
+arram mandel
 
 
-
-## Name of Session: 
+## Name of Session:
+disrupting your perspective: become an (secret) agent of disruption
```

Comment:
Students might need more than this for the description...become an agent of disruption and take control of your future (or something like that?)